test/Microsoft.TestPlatform.CommunicationUtilities.PlatformTests/SocketClientTests.cs

```diff
@@ -38,9 +38,11 @@ public void Dispose()
 #if NETFRAMEWORK
             // tcpClient.Close() calls tcpClient.Dispose().
             this.tcpClient?.Close();
+            GC.SuppressFinalize(this);
 #else
             // tcpClient.Close() not available for netcoreapp1.0
             this.tcpClient?.Dispose();
+            GC.SuppressFinalize(this);
```

Comment:
nit: Move this after `#endif`.

Reply:
resolved added GC.SuppressFinalize(this); after the #endif